src/BenchmarkDotNet.Core/Extensions/CommonExtensions.cs

```diff
@@ -20,24 +17,34 @@ public static List<T> ToSortedList<T>(this IEnumerable<T> values)
             return list;
         }
 
-        public static string ToTimeStr(this double value, TimeUnit unit = null, int unitNameWidth = 1)
+        public static string ToTimeStr(this double value, TimeUnit unit = null, int unitNameWidth = 1, bool showUnit = true)
         {
             unit = unit ?? TimeUnit.GetBestTimeUnit(value);
             var unitValue = TimeUnit.Convert(value, TimeUnit.Nanosecond, unit);
-            var unitName = unit.Name.PadLeft(unitNameWidth);
-            return $"{unitValue.ToStr("N4")} {unitName}";
+            if (showUnit)
+            {
+                var unitName = unit.Name.PadLeft(unitNameWidth);
+                return $"{unitValue.ToStr("N4")} {unitName}";
+            }
+            else
+            {
+                return $"{unitValue.ToStr("N4")}";
+            }
         }
 
-        internal static string ToFormattedBytes(this long bytes)
+        public static string ToSizeStr(this long value, SizeUnit unit = null, int unitNameWidth = 1, bool showUnit = true)
```

Comment:
We could do it, and I would rather use less extension methods, but then we'd split the "to string" functionality into `ToStr` in `CommonExtensions`, `SizeUnit` and `TimeUnit`. Now they are at least all in the same location. What do you think?